lib/train/utils.py

```diff
@@ -319,6 +319,9 @@ def get_hparams(init=True):
     parser.add_argument(
         "-e", "--experiment_dir", type=str, required=True, help="experiment dir"
     )  # -m
+    parser.add_argument(
+        "-exp", "--exp_dir", type=str, required=True, help="export dir"

```

Comment:
Be verbose, at least in the full name (`--export_dir`, not to be confused with, e.g. `--experiment_dir`).

Reply:
one